pyproject.toml

```diff
@@ -77,7 +77,7 @@ slack-sdk = "^3.26.2"
 joblib = "^1.3.2"
 
 [tool.poetry.group.wizard.dependencies]
-streamlit-nightly = "^1.35.1.dev20240610"
+streamlit-nightly = "1.35.1.dev20240610"
```

Comment:
Intentional?

Reply:
.@lucasrodes added that one because of problems in Streamlit versions. It is by owidbot so it should be fine (Lucas, let us know otherwise)